test/test_errors.py

```diff
@@ -85,10 +85,10 @@ def test_handle_result_error(container_factory, rabbit_config):
             # proxy.task() will hang forever because it generates an error
             proxy.task.call_async()
 
-        with eventlet.Timeout(1):
```

Comment:
hm. looking at travis, builds historically take ~3 minutes. this pr is around 9. worth checking if this is travis having issues or this pr